lib/rake_compiler_dock.rb

```diff
@@ -58,11 +58,16 @@ def exec(*args)
     runargs.unshift("sigfw") if options.fetch(:sigfw){ true }
     runargs.unshift("runas") if options.fetch(:runas){ true }
 
-    if RUBY_PLATFORM =~ /mingw|mswin/
+    case RUBY_PLATFORM
+    when /mingw|mswin/
       # Change Path from "C:\Path" to "/c/Path" as used by boot2docker
       pwd = Dir.pwd.gsub(/^([a-z]):/i){ "/#{$1.downcase}" }
       uid = 1000
       gid = 1000
+    when /darwin/
+      pwd = Dir.pwd
+      uid = 1000
+      gid = Process.gid
```

Comment:
Thank you for working on this! I would like to understand the change. Why is the uid set to a fixed value but the git of the host system?